src/Components/DetailPage/DetailPage.js

```diff
@@ -0,0 +1,191 @@
+import React, { useState, useEffect } from "react";
+import DetailProps from "./DetailProps";
+import styled from "styled-components";
+import Svg from "../DetailPage/DetailPageSvg";
+
+const DetailPage = () => {
+  const [data, setData] = useState([]);
+  useEffect(() => {
+    fetch("http://127.0.0.1:8000/positions/106")
+      .then((res) => res.json())
+      .then((res) => setData([res.detail]));
+  }, []);
+
+  return (
+    <DetailPageContainer>
+      <DetailBox>
+        <DetailPageMain>
+          {data &&
+            data.map((el, idx) => (
+              <DetailProps {...el} key={el[idx]} img={el.img[0]} />
+            ))}
+        </DetailPageMain>
+        <DetailPageSub>
+          <DetailPageSubBox>
+            <CompensationMoney>채용보상금</CompensationMoney>
+            <ShareBtn>{Svg.ShaerIcon}</ShareBtn>
+            <ContentBox>
+              <RecommendedPerson>추천인</RecommendedPerson>
```

Comment:
```suggestion
              <Person>추천인</Person>
```
- 이름이 조금 기네요. 줄여주면 좋겠습니다.
- 추천인에는 해당 이름이 어울리는데 지원자에는 어울리지 않는 것 같습니다.
- Person보다 좋은 이름이 있다면 제안해주세요!